src/modules/dashboard/components/building-list.vue

```diff
@@ -111,6 +112,22 @@
                 <td v-if="listType === 'extension'">
                     {{ building.extension_unavailable.toLocaleString() }}
                 </td>
+                <td
+                    v-if="
+                        listType === 'building' &&
+                        bedBuildingsType.includes(building.building_type)
+                    "
+                >
+                    {{ building.level + 10 }}
```

Comment:
please use the `startBeds` from building translations instead of 10 as e.g. the `clinic` building on en_GB initially has 5 beds:
https://github.com/LSS-Manager/LSSM-V.4/blob/af1b1137cac01636a1bc697e80e641cd4d91d297/typings/Building.d.ts#L112